### PR TITLE
made product page

### PR DESCRIPTION
**Pull request by cjinc121**
- made products page

### CODE REVIEW

File: screens/product/product.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+.text-end {
+  text-align: end;
+}
+.sidebar {
+  background-color: var(--white-color);
+  width: 20%;
+  position: relative;
+}
+.main-heading {
+  margin: 0.1rem;
```
Comment (ArjunGTX) [inline screens/product/product.css]: try to use pixels if values are too small

File: screens/product/product.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    letter-spacing: 2px;
+    display: flex;
+    font-size: 2rem;
+    width: 100%;
+    position: fixed;
+    z-index: 1;
+    bottom: 0;
+    border-top: 2px solid black;
+    margin-top: 1rem;
+    padding: 1rem 0;
```
Comment (ArjunGTX) [inline screens/product/product.css:107]: since `1rem` is used in many places, it is better to store it in a variable and reuse

**Review by ArjunGTX — APPROVED** [review]

**Comment by ArjunGTX** [comment]
Everything else looks good

File: screens/product/product.css (hunk 2 above)
Comment (cjinc121) [reply screens/product/product.css:107]: would keep this in mind

File: screens/product/product.css (hunk 1 above)
Comment (cjinc121) [reply screens/product/product.css]: okay

File: screens/product/product.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+.sidebar-heading {
+  font-weight: bold;
+  letter-spacing: 2px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.radio input {
+  margin-right: 0.3rem;
```
Comment (ShreyashMahajan) [inline screens/product/product.css]: use px for less than 1rem

**Review by ShreyashMahajan — APPROVED** [review]
code looks good and well structured. Good to merge

**Pull request merged**